### PR TITLE
Images

### PR DESCRIPTION
**Pull request by FomichevaEkaterina**
https://htmlacademy.ru/courses/basic-html-css/homeworks/13/steps/1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,14 +3,20 @@
...
     <title>Портфолио Junior верстальщика</title>
   </head>
 
   <body>
     <header class="page-header">
       <div class="container">
-        Judie's home
+        <a class="header-email" href="katarina.santalova@gmail.com"
+          >katarina.santalova@gmail.com</a
+        >
```
Comment (gfomichev) [inline index.html]: What happened to the markup?

File: index.html [hunk 2]
```diff
@@ -39,19 +53,30 @@ <h3>Чем могу быть полезен</h3>
...
         <p>Статический блог про обучение HTML и CSS.</p>
       </div>
     </section>
 
     <footer class="page-footer">
       <div class="container">
-        Звоните Космосу.
+        <a class="footer-email" href="katarina.santalova@gmail.com"
+          >katarina.santalova@gmail.com</a
+        >>
```
Comment (gfomichev) [inline index.html]: What happened to the markup? #2

File: index.html [hunk 3]
```diff
@@ -3,14 +3,20 @@
...
+      rel="stylesheet"
+    />
     <title>Портфолио Junior верстальщика</title>
   </head>
 
   <body>
     <header class="page-header">
       <div class="container">
-        Judie's home
+        <a class="header-email" href="katarina.santalova@gmail.com"
```
Comment (gfomichev) [inline index.html]: href shoud include mailto: prefix

File: index.html [hunk 4]
```diff
@@ -3,14 +3,20 @@
...
+          >katarina.santalova@gmail.com</a
+        >
+        <a class="header-phone" href="+7 964 794-53-65">+7 964 794-53-65</a>
```
Comment (gfomichev) [inline index.html]: href should include tel: prefix and should not include whitespaces and hyphens

File: index.html [hunk 5]
```diff
@@ -39,19 +53,30 @@ <h3>Чем могу быть полезен</h3>
 
     <section class="portfolio">
       <div class="container">
+        <figure class="project-image">
```
Comment (gfomichev) [inline index.html]: project image should be placed **under** subheading

File: index.html [hunk 6]
```diff
@@ -23,6 +29,14 @@ <h1 class="heading">Екатерина Фомичева</h1>
 
     <section class="intro">
       <div class="container">
+        <figure class="user-image">
```
Comment (gfomichev) [inline index.html]: user image should be placed **under** subheading

**Review by gfomichev — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (FomichevaEkaterina) [reply index.html]: That was VS Code plugin issue, fixed

File: index.html (hunk 2 above)
Comment (FomichevaEkaterina) [reply index.html]: That was VS Code plugin issue, fixed

File: index.html (hunk 3 above)
Comment (FomichevaEkaterina) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (FomichevaEkaterina) [reply index.html]: fixed

File: index.html (hunk 6 above)
Comment (FomichevaEkaterina) [reply index.html]: fixed

File: index.html (hunk 5 above)
Comment (FomichevaEkaterina) [reply index.html]: fixed

**Review by gfomichev — APPROVED** [review]

**Pull request merged**